Description: don't create dummy files in log dir on package build
Author: Marc Haber <mh+debian-packages@zugschlus.de>
Forwarded: no
Last-Update: 2016-08-07
--- a/Makefile
+++ b/Makefile
@@ -97,8 +97,6 @@ sysvinstall:	genericinstall
 		chmod 0711        $(DESTDIR)$(SCRPATH)/atop.daily
 		cp psaccs_atop    $(DESTDIR)$(ROTPATH)/psaccs_atop
 		cp psaccu_atop    $(DESTDIR)$(ROTPATH)/psaccu_atop
-		touch             $(DESTDIR)$(LOGPATH)/dummy_before
-		touch             $(DESTDIR)$(LOGPATH)/dummy_after
 		#
 		if [   -d $(DESTDIR)$(PMPATH1) ]; 			\
 		then	cp 45atoppm $(DESTDIR)$(PMPATH1); 		\
